src/qurveros/optspacecurve.py

```diff
@@ -42,7 +42,6 @@ class OptimizableSpaceCurve(spacecurve.SpaceCurve):
         When an instance is indexed with a string, optimization information
         is returned. See the optimize method.
     """
-
```

Comment:
Please fix this removed newline- this file should not have any changes if all of the logic is moved to `spacecurve.py`.